cli/lib/kontena/cli/apps/restart_command.rb

```diff
@@ -27,10 +27,11 @@ def execute
     def restart_services(services)
       services.each do |service_name, opts|
         if service_exists?(service_name)
-          puts "restarting #{prefixed_name(service_name)}"
-          restart_service(token, prefixed_name(service_name))
+          ShellSpinner "sending restart signal to #{service_name.colorize(:cyan)} " do
+            restart_service(token, prefixed_name(service_name))
+          end
         else
-          puts "No such service: #{service_name}".colorize(:red)
+          STDERR.puts "WARNING: no such service: #{service_name}".colorize(:yellow)
```

Comment:
I don't think this is a "warning", isn't this "I have not done anything, aboring the whole thing", I think it should be in red and a "ERROR" + give exit code other than 0 (and use the abort() method?)

`kontena app restart && echo "successfully restarted!"` should not happen if the restart fails